### ISSUE TITLE
Decorator `@` is stripped if used after `// prettier-ignore`

### ISSUE THREAD

**Issue by paustint** [dangmai/prettier-plugin-apex#383]
There are two bugs I found.

What I wanted to do: I did not like how `prettier-apex` formatted my enum on a new line for each member because my enum only had two members, so I used `// prettier-ignore` and all was fine until I tried to add @TestVisible to a private enum, then all 🔥 hell🔥  broke loose.

I am not 100% sure if comments are allowed between decorators and the member the decorator is applying to, but I resorted to trying that after `BUG 1`, which led me to identify `BUG 2`.

```sh
# Prettier options (if any):
{
  "trailingComma": "none",
  "apexInsertFinalNewline": false,
  "singleQuote": true,
  "printWidth": 140,
  "overrides": [
    {
      "files": "**/classes/**/*.cls",
      "options": { "tabWidth": 4 }
    },
    {
      "files": "**/lwc/**/*.html",
      "options": { "parser": "lwc", "tabWidth": 4 }
    },
    {
      "files": "*.{cmp,page,component}",
      "options": { "parser": "html", "tabWidth": 4 }
    }
  ]
}

```

**Input:**

```apex
// BUG 1
// prettier-ignore
@TestVisible
private enum AllOrAny { ALL, ANY }

OR

// BUG 2
@TestVisible
// prettier-ignore
private enum AllOrAny { ALL, ANY }
```

**Actual output:**

```apex
// BUG 1
// prettier-ignore
TestVisible
private enum AllOrAny { ALL, ANY }

// Bug 2
@TestVisible
// prettier-ignore
privateenum AllOrAny {
    ALL,
    ANY
}
```

**Expected output:**

```apex
// !BUG 1
// prettier-ignore
@TestVisible
private enum AllOrAny { ALL, ANY }

// !Bug 2
@TestVisible
// prettier-ignore
private enum AllOrAny { ALL, ANY }
```

**Additional information (please fill this out)**:

- OS: OSX
- Version: 
  - `"prettier-plugin-apex": "^1.9.1"`
  - `"prettier": "^2.3.0",`

**Comment by dangmai** [comment]
That's indeed an issue then, thank you for opening the bug report. I'll take a look and fix it soon.

**Comment by dangmai** [comment]
Fixed in `master`, I'm also releasing `1.10.0` that will include it. Thanks again!
